src/main/java/com/github/dockerjava/api/model/Node.java

```diff
@@ -8,20 +9,30 @@
  * A node as returned by the /events API, for instance, when Swarm is used.
  */
 @JsonInclude(Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class Node {
 
     @JsonProperty("Name")
     private String name;
 
-    @JsonProperty("Id")
+    @JsonProperty("ID")
```

Comment:
I don't remember link, but one of users that tried to dig into complain that swarm changes everything in every version. 
So this would be the question whether swarm versioning his own API (that is of course on top of docker, but still it's API with changes/versions?).